rent_golem/backend/rent_golem/clusters/tests.py

```diff
@@ -1,3 +1,60 @@
-from django.test import TestCase
+from django.shortcuts import reverse
 
-# Create your tests here.
+from rest_framework.test import APIClient, APITestCase, APIRequestFactory
+from rest_framework import status
+
+
+class ClustersViewSetTest(APITestCase):
+    def setUp(self) -> None:
+        self.factory = APIRequestFactory()
+        self.client = APIClient()
+
+    def create_dummy_db_object(self):
+        json_data = {
+            "uuid": "dcec5482-59ef-4c0e-9ea4-4e5451c3cbda",
+            "package_type": "automatic",
+            "status": "Starting",
+            "additional_params": {},
+            "size": 5,
+        }
+        return self.client.post(reverse('clusters-list'), data=json_data, format="json"), json_data
+
+    def test_post_one_object_to_db(self):
+        response, json_data = self.create_dummy_db_object()
+        self.assertEquals(response.status_code, status.HTTP_201_CREATED)
+
+    def test_read_one_object_from_db(self):
+        post_response, json_data = self.create_dummy_db_object()
+
+        get_response = self.client.get(reverse('clusters-detail', args=[json_data["uuid"]]))
+        self.assertEquals(get_response.status_code, status.HTTP_200_OK)
+        for keyword in json_data.keys():
+            self.assertEquals(json_data[keyword], get_response.data[keyword])
```

Comment:
Again, let's create the object directly through ORM and then try to fetch it.
Better variable names would be helpful.

For example:
```
def test_read_one_object_from_db(self):
        cluster = Cluster.objects.create(...)

        response = self.client.get(reverse('clusters-detail', args=(cluster.id, )))
        self.assertEquals(response.status_code, status.HTTP_200_OK)
        self.assertEquals(response['uuid'], cluster.uuid)
        self.assertEquals(response['status'], cluster.status)
        ...
```